From: Robert Schwebel <r.schwebel@pengutronix.de>
Date: Sat, 29 Oct 2011 18:46:37 +0200
Subject: [PATCH] compiling without pam doesn't work

When compiled with --with-authfw=none, I get

polkit-resolve-exe-helper.c:46:31: error: security/pam_appl.h: No such file or directory

The patch below fixes it.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
---
 src/polkit-dbus/polkit-resolve-exe-helper.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/polkit-dbus/polkit-resolve-exe-helper.c b/src/polkit-dbus/polkit-resolve-exe-helper.c
index c56b2f5..cf696a9 100644
--- a/src/polkit-dbus/polkit-resolve-exe-helper.c
+++ b/src/polkit-dbus/polkit-resolve-exe-helper.c
@@ -43,7 +43,9 @@
 #ifdef HAVE_FREEBSD
 #include <sys/param.h>
 #endif
+#ifdef HAVE_PAM
 #include <security/pam_appl.h>
+#endif
 #include <grp.h>
 #include <pwd.h>
 #include <syslog.h>
